lib/src/codecs/codec_IMUEvent.cpp

```diff
@@ -86,35 +86,34 @@ std::string IMUevent::getType() const
     return IMUevent::tag;
 }
 
-double IMUevent::convertToSI(int index, int value)
+double IMUevent::convertToSI(int value, int index)
 {
-    static constexpr double ay_bias = 0.12;
     static constexpr double ax_bias = 0.21;
+    static constexpr double ay_bias = 0.12;
     static constexpr double az_bias = 0.06;
-    static constexpr double ay_gain = 9.80665/9.83;
     static constexpr double ax_gain = 9.80665/9.84;
+    static constexpr double ay_gain = 9.80665/9.83;
     static constexpr double az_gain = 9.80665/9.95;
 
 
     switch(index) {
     case(ACC_X):
```

Comment:
sorry Leandro - there is still a mistake. ACC_X has its own biases and therefore has it's own equation. The comparison against master should only show updates to the order of the arguments and where the minus signs are placed.